src/main/java/com/limechain/trie/structure/StorageNodeHandle.java

```diff
@@ -13,4 +13,16 @@ public final class StorageNodeHandle<T> extends NodeHandle<T> {
     public boolean hasStorageValue() {
         return true;
     }
+
+    public boolean clearStorageValue() {
+        return trieStructure.clearNodeValue(rawNodeIndex);
+    }
+
+    //convert to branchNode
```

Comment:
you can delete this, no point in commenting if the function name is going to give the same info